roles/common/tasks/main.yml

```diff
@@ -82,7 +87,16 @@
     owner: "{{ omit if archive_owner == '' else archive_owner }}"
     mode: 0755
     creates: "{{binary_base_path}}"
-  when: installation_method == "archive"
+  when:
+    - installation_method == "archive"
+    - ansible_os_family != "OS/390"
+
+- name: Expand remote Confluent Platform archive (z/OS)
```

Comment:
This is a good improvement but its diverging from existing structure. I would like to take this up in separate PR and probably implement for all OS

Reply:
Existing structure just wouldn't work for z/OS, because it is way too special :D